scripts/jenkins/cloud/manual/configure-deployer-container.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Create deployer container
+  hosts: ci-deployer
+  connection: docker
+  remote_user: root
+  gather_facts: False
+  tasks:
+    - name: Read cloud config files
+      set_fact:
+        clouds: "{{ lookup('file', item) | from_yaml | combine(clouds | default({}), recursive=True) }}"
+      with_fileglob:
+        - "{{ lookup('env', 'HOME') + '/.config/openstack/*' }}"
+
+    - name: Replace cacert paths
+      set_fact:
+        clouds: "{{ clouds | combine(cacert, recursive=True) }}"
+      vars:
+        cacert: "{ 'clouds': { '{{ item }}': { 'cacert': '/usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem' } } }"
+      with_items: |
+        [ {% for cloud in clouds['clouds'] %} '{{ cloud }}', {% endfor %} ]
```

Comment:
if cloud['clouds'] is a list do you need the for loop? or can you do: 
"with_items: clouds['clouds']"

(I don't know for sure)

Reply:
This is definitely tricky :smile: I have replace this functionality with the entrypoint script now. I have to admit though that the new code has about 100 times as many lines of code :wink: